src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImp registrationService;
+    private final StorageDaoImpl storageMock;
+
+    RegistrationServiceImplTest(StorageDaoImpl storageMock) {
+        this.storageMock = storageMock;
+    }
```

Comment:
You should not be injecting the `StorageDaoImpl` through the constructor in a test class. Instead, you should be mocking this object, for example, using Mockito. This is because in unit tests, you want to isolate the class you're testing from its dependencies.